types/react-native/v0.63/index.d.ts

```diff
@@ -6901,7 +6901,7 @@ export class SwipeableListView extends React.Component<SwipeableListViewProps> {
 export interface ActionSheetIOSOptions {
     title?: string;
     options: string[];
-    cancelButtonIndex?: number;
+    cancelButtonIndex?: number | number[];
     destructiveButtonIndex?: number;
```

Comment:
I think this is a mistake here and should be the other way, right?

```suggestion
    cancelButtonIndex?: number;
    destructiveButtonIndex?: number | number[] | undefined | null;
```